client/file_logging.py

```diff
@@ -0,0 +1,138 @@
+import king_phisher.client.plugins as plugins
+import king_phisher.client.gui_utilities as gui_utilities
+
+try:
+	import logging
```

Comment:
logging is a base import of python, please move it above the imports of king phisher